test/unit/README.md

```diff
@@ -3,40 +3,105 @@
 
 ### Running unit tests
 
-1. via the command line
+1. Running just unit tests - `npm run test`
 
-```
-grunt test
-```
+2. Running coverage reporter - `npm run coverage`. 
+To check coverage, open `./coverage/lcov-report/index.html`.
 
-2. in the browser
-```
-open SpecRunner.html
-```
+3. Running tests in browser - `test-browser`.
 
 
-### Adding a new test file
+### Common concepts for writing tests
 
-New test files are added as a script tag to the SpecRunner.html
+1. Test suites should cover only public API.
 
-```html
-<script src="test/unit/application.app-regions.spec.js"></script>
-<script src="test/unit/application.spec.js"></script>
-```
+2. Code style.
 
+- Each `describe` should have name of tested method. 
+
+> There are should be not more then 1 nested describe for on method.
```

Comment:
```diff
- There are should
+ There should
```
I'm ok with starting here, but I do wonder if at some point we'll need to allow for a third nest.  I'd like to think not, but there might be the case for one somewhere to keep from duplication overkill.  But I very much agree that shallow nesting is the goal.